README.md

```diff
@@ -23,145 +23,165 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
 
 You can check if these packages are installed and functioning by running:
 ```
-git version
-java -version
+docker version
 docker-compose version
 docker run hello-world
 ```
 
-<details>
-  <summary>Installing Java...</summary>
-
-  Deephaven can be built with either [Oracle JDK](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html)
-   or [OpenJDK](https://openjdk.java.net/install/).  Java 8 is required.  To install Java, run
-  * Mac
-      ```
-      brew install openjdk@8
-      ```
-      OpenJDK 8 may need to be added to your path:
-      ```
-      echo 'export PATH="/usr/local/opt/openjdk@8/bin:$PATH"' >> ~/.zshrc
-      ```
-  * Windows WSL2 - Ubuntu
-      ```
-      sudo apt update
-      sudo apt install openjdk-8-jdk-headless
-      ```
-  * Linux
-      ```
-      sudo apt update
-      sudo apt install openjdk-8-jdk
-      ```
-    or
-      ```
-      sudo yum install java-1.8.0-openjdk
-      ```
-</details>
-
-<details>
-  <summary>Installing Docker...</summary>
-
-  Instructions for installing and configuring Docker can be found at
-  [https://docs.docker.com/get-docker/](https://docs.docker.com/get-docker/).  Windows users should follow the WSL2 instructions.
-  Docker Engine Version 18.03 (or newer) is needed to build.
-  Docker Engine Version 19.03 (or newer) is needed to build and run.
- 
-
-  Instructions for installing and configuring `docker-compose` can be found at
-  [https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/).
-  Version 1.29 (or newer) is recommended; Version 1.27 (or newer) is required.
-</details>
-
-<details>
-  <summary>Docker RAM settings...</summary>
-
-  Tests run as part of the build process require at least 4GB of Docker RAM.  To check your Docker configuration, run:
-  ```
-  docker info | grep Memory
-  ```
-
-  By default, Docker on Mac is configured with 2 GB of RAM.  If you need to increase the memory on your Mac, click
-  on the Docker icon on the top bar and navigate to `Preferences->Resources->Memory`.
-
-  ![alt_text](docs/images/DockerConfigMac.png "Docker Configuration on a Mac")
-</details>
-
-<details>
-  <summary>If <code>docker run hello-world</code> does not work...</summary>
-
-  If `docker run hello-world` does not work, try the following:
-  1. [Is Docker running?](https://docs.docker.com/config/daemon/#check-whether-docker-is-running)
-      ```
-      docker info
-     ```
-  2. (Linux) [Are you in the `docker` user group?](https://docs.docker.com/engine/install/linux-postinstall/)
-      ```
-      sudo groupadd docker
-      sudo usermod -aG docker $USER
-      ```
-</details>
-
-
-### Checkout & Build Deephaven
-
-Once all of the required dependencies are installed and functioning, run:
+On Windows, these commands must be run using WSL 2.
+
+If any dependencies are missing or unsupported versions are installed, 
+see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/launch-pre-built).
```

Comment:
```suggestion
see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/launch-pre-built/#prerequisites) for installation instructions.
```
Small wording change to emphasize that installation instructions are here. Also linking to the specific header may make it more clear

Reply:
also remove the extra line break, please